app/assets/stylesheets/_colors.css.scss

```diff
@@ -0,0 +1,21 @@
+// Defines color variables for reuse
+$bs-default-primary: darken(#428bca, 6.5%);
```

Comment:
Color literals like `#428bca` should only be used in variable declarations; they should be referred to via variable everywhere else.